src/components/CodePopover/CodePopover.tsx

```diff
@@ -49,12 +51,14 @@ export const CodePopover: FunctionComponent<CodePopoverProps> = ({
   const { hasCopied, onCopy } = useClipboard(code);
   const disclosure = useDisclosure();
   const focusRef = useRef<HTMLButtonElement>(null);
+  const mdBreakpoint = useToken("breakpoints", "md");
+  const [isMd] = useMediaQuery(`(min-width: ${mdBreakpoint})`);
 
   return (
     <Popover
       initialFocusRef={focusRef}
       isLazy
-      placement="bottom-end"
+      placement={isMd ? "bottom-end" : "bottom"}
```

Comment:
This change is not a bug fix, just centers the code popover with it's trigger on mobile